src/routes/tags.js

```diff
@@ -0,0 +1,81 @@
+const tagRouter = require('express').Router()
+const db = require('../db')
+
+// FETCH ALL PREDEFINED TAGS
+
+const fetchTags = async (_req, res) => {
+    try {
+        const tags = await db.query('SELECT * FROM tags')
+        res.status(200).json(tags.rows)
+    } catch (error) {
+        console.error(error)
+        res.status(500).json({ message: 'Internal server error', error })
+    }
+}
+
+// CREATE NEW TAG
+
+const createTag = async (req, res) => {
+    try {
+        const { content, title } = req.body
+
+        const newTag = await db.query(
+            'INSERT INTO tags (content, title) SELECT $1, $2 WHERE NOT EXISTS (SELECT * FROM tags WHERE title = $2) RETURNING *',
+            [content, title]
+        )
+
+        if (!newTag.rows[0]) {
+            res.status(400).json({ message: 'Tag already exists' })
+        } else {
+            res.status(202).json({
+                message: 'Successfully created a new tag',
+                data: newTag.rows[0],
```

Comment:
let's kindly define what we want to return here as well as you've done for the other one. Kindly do the same for other ones if possible. Thanks

Reply:
Resolved